api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -47,22 +48,39 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
-        String locationUserPropertyName = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME);
-        if(StringUtils.isBlank(locationUserPropertyName)) {
-            Context.getAdministrationService().setGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
-                    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
-            if(log.isDebugEnabled()) {
-                log.debug("RefApp Location global property created with value - " + LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
-            }
-        }
-        else if(StringUtils.isNotBlank(locationUserPropertyName) && !locationUserPropertyName.equals(LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME)) {
-            log.warn("RefApp Location global property already exist in the system with different value(" + locationUserPropertyName +
-                    "). Exiting from creating new global property with the value " + LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
-        }
-        log.info("Location Based Access Control Module started");
+	    PersonAttributeType personAttributeType = getPersonAttributeType();
```

Comment:
Will, it always creates a personAttributeType when the module starts?

Reply:
I Should change this, as  running the server again and again the it shows the error `Duplicate entry '0a93cbc6-5d65-4886-8091-47a25d3df944' for key 'person_attribute_type_uuid_index'`. so we need to add only if it is not found.